backend-worker/worker.py

```diff
@@ -89,78 +86,25 @@
 PIXELSIZE_X, PIXELSIZE_Y = _NLCD_RASTER_INFO['pixel_size']
 NLCD_COLORS = {}  # update this dict later in file once function is defined.
 
-INVEST_SAMPLE_DATA = 'invest-sample-data'
-INVEST_BASE_PATHS = {
-    'docker': f'/opt/appdata/{INVEST_SAMPLE_DATA}',
-    'local': os.path.join(
-        os.path.dirname(__file__), '..', 'appdata', INVEST_SAMPLE_DATA)
-}
-INVEST_BASE_PATH = None
-for data_path in INVEST_BASE_PATHS.values():
-    if os.path.exists(data_path):
-        INVEST_BASE_PATH = data_path
-        break
-if INVEST_BASE_PATH is None:
-    raise AssertionError(
-        f"Could not find {INVEST_SAMPLE_DATA} at any known locations")
-LOGGER.info(f"Using InVEST data at {INVEST_SAMPLE_DATA}")
-
+CARBON = 'carbon'
+URBAN_COOLING = 'urban_cooling_model'
 INVEST_MODELS = {
-    "pollination": {
-        "api": pollination,
-        "data_key": "pollination", 
-        "args_path": os.path.join(
-            INVEST_BASE_PATH, 'pollination', 'pollination_args.json')},
-    "stormwater": {
-        "api": stormwater,
-        "data_key": "UrbanStormwater",
-        "args_path": os.path.join(
-            INVEST_BASE_PATH, 'UrbanStormwater', 'urban_stormwater_args.json')},
-    "urban_cooling_model": {
+    URBAN_COOLING: {
         "api": urban_cooling_model,
-        "data_key": "UrbanCoolingModel",
-        "args_path": os.path.join(
-            INVEST_BASE_PATH, 'UrbanCoolingModel',
-            'urban_cooling_model_args.json')},
-    "carbon": {
+        "build_args": invest_args.urban_cooling
+    },
+    CARBON: {
         "api": carbon,
-        "data_key": "Carbon",
-        "args_path": os.path.join(
-            INVEST_BASE_PATH, 'Carbon', 'carbon_args.json')},
-    "urban_flood_risk_mitigation": {
-        "api": urban_flood_risk_mitigation,
-        "data_key": "UrbanFloodMitigation",
-        "args_path": os.path.join(
-            INVEST_BASE_PATH, 'UrbanFloodMitigation',
-            'urban_flood_mitigation_args.json')},
-    "urban_nature_access": {
-        "api": urban_nature_access,
-        "data_key": "UrbanNatureAccess",
-        "args_path": os.path.join(
-            INVEST_BASE_PATH, 'UrbanNatureAccess',
-            'urban_nature_access_args.json')},
+        "build_args": invest_args.carbon
+    }
 }
 LARGEST_SERVICESHED = 2230  # meters https://github.com/natcap/urban-online-workflow/issues/79
 
 # Quiet logging
 logging.getLogger(f'pygeoprocessing').setLevel(logging.WARNING)
-logging.getLogger(f'natcap.invest').setLevel(logging.WARNING)
+# logging.getLogger(f'natcap.invest').setLevel(logging.WARNING)
```

Comment:
Should this be left commented or removed?

Reply:
I removed it. It's being handled by `utils.prepare_workspace` for now. Easy to add it back as needed.